src/CMakeLists.txt

```diff
@@ -35,19 +51,21 @@ if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
       CMAKE_C_FLAGS_${build_type_upper})
     string(REGEX REPLACE "(^| )[/-]D *NDEBUG($| )" " " "${flags}" "${${flags}}")
   endforeach()
-endif()
+endif(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
```

Comment:
yes, 1) but i don't think we need to "fix" it. the `if` block is smaller enough, that we know where it starts. 2) and you are not changing the `if` block, so i don't really think you were there.